netket/hilbert/random/continuous.py

```diff
@@ -0,0 +1,21 @@
+import jax
+from jax import numpy as jnp
+
+from netket.hilbert import Particles
+from netket.utils.dispatch import dispatch
+
+
+@dispatch
+def random_state(hilb: Particles, key, batches: int, *, dtype):
+    """Positions particles w.r.t. normal distribution if no periodic boundary conditions are applied in a spatial dimension.
+    Otherwise the particles are positioned evenly along the box from 0 to L, with Gaussian noise."""
```

Comment:
can you try to have lines not crazy long? 
We don't enforce the 89 character limit on docstrings, but it's better if we loosely try to abide by it when possible...
x